ide/css.lib/src/org/netbeans/modules/css/lib/nblexer/NbLexerCharStream.java

```diff
@@ -132,9 +132,9 @@ public void release(int marker) {
         }
         
         //remove all markers from the given one, including the requested one
-        for(int i = marker; i < markers.size(); i++) {
-            markers.remove(i);
-        }
+        do {
+            markers.remove(marker);
+        } while (marker < markers.size());
```

Comment:
> Independent of the implementation of the stack I would expect the `push` and `pop` operations to be the fastest operations. So instead of using the positional remove I'd use `markers.pop()` to remove the superfluous markers.

good point, changed as suggested